crypto/threads_pthread.c

```diff
@@ -655,6 +842,20 @@ int CRYPTO_THREAD_unlock(CRYPTO_RWLOCK *lock)
 # ifdef USE_RWLOCK
     if (pthread_rwlock_unlock(lock) != 0)
         return 0;
+#  ifdef REPORT_RWLOCK_CONTENTION
+    {
```

Comment:
technically this block parren should not be needed for C-99 code. but we have not codified a style yet...